Classes/Controller/AbstractController.php

```diff
@@ -176,6 +176,27 @@ protected function loadDocument(int $documentId = 0)
         }
     }
 
+    /**
+     * Configure URL for proxy.
+     *
+     * @access protected
+     *
+     * @param string $url URL for proxy configuration
+     *
+     * @return void
+     */
+    protected function configureProxyUrl(&$url) {
+        $this->uriBuilder->reset()
+            ->setTargetPageUid($GLOBALS['TSFE']->id)
+            ->setCreateAbsoluteUri(!empty($this->settings['forceAbsoluteUrl']) ? true : false)
```

Comment:
`empty()` already returns `true` or `false`, so we can shorten this a little bit.

```suggestion
            ->setCreateAbsoluteUri(!empty($this->settings['forceAbsoluteUrl']))
```